### PR TITLE
Create Controller.java

### PR DESCRIPTION
**Pull request by PravinKumarRajaSahayam**
I created new branch and added Controller.java file in Git Repository.
Should you review my code?

### CODE REVIEW

**Comment by Gayathrik05** [comment]
I think, you forgot to add view all method in controller file. Can you add that method?

File: Controller.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    {
+ 	   dao.save(vms);
+	return "Updated";
+    }
+	@DeleteMapping("/PerformDelete/{laptop_id}")
+    public String performDelete(@PathVariable("laptop_id") int laptopid)
+    {
+ 	   dao.deleteById(laptopid);
+	return "Deleted";
+    }
```
Comment (PravinKumarRajaSahayam) [inline Controller.java:38]: I updated my Controller.java with adding Find All method file.Could you recheck my file

**Review by PravinKumarRajaSahayam — COMMENTED** [review]
I updated my Controller.java with adding Find All method file.Could you recheck my file

**Review by Gayathrik05 — COMMENTED** [review]
Okay . Now you can merge your files.

**Pull request merged**